blazor/heatmap-chart/events.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: Events in Blazor HeatMap Component | Syncfusion
+description: Checkout and learn here all about events in Syncfusion Blazor HeatMap component and much more.
+platform: Blazor
+control: HeatMap Chart
+documentation: ug
+---
+
+# Events in Blazor HeatMap chart Component
+ 
+This section explains the list of events that will be triggered for appropriate actions in HeatMap. The events should be provided to the HeatMap using the [HeatMapEvents](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.HeatMap.HeatMapEvents.html).
+
+## CellRender
+
+When the [CellRendering](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.HeatMap.HeatMapEvents.html#Syncfusion_Blazor_HeatMap_HeatMapEvents_CellRendering) event is triggered on rendering for each cell in the Heatmap component. To know more about the arguments of this event, refer [here](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.HeatMap.HeatMapCellRenderEventArgs.html).
```

Comment:
Use the same content as in this page https://ej2.syncfusion.com/documentation/heatmap-chart/events#cellrender